episodes/designing-a-light-microscopy-experiment.md

```diff
@@ -0,0 +1,487 @@
+section of this episode](#widefield)). The downside is that it's much harder to 
+automatically recognise and measure nuclei from these kinds of images. Unlike 
+images of fluorescently labelled nuclei, there is a much less clear separation 
+between nuclei and the rest of the cell, making analysis more challenging.
+
+### Spatial scale
+
+We need to observe many nuclei at once, to get a good estimate of their number 
+and average shape and size. Therefore, our spatial scale should encompass an 
+area large enough to image many nuclei for each timepoint (say 100 nuclei).
+
+### Timescale
+
+As we are interested in how the nuclei number, shape and size changes over time, 
+we will need to use live cell imaging. We will need to image for long enough to 
+see multiple cell divisions, so we can assess how the number of nuclei increases. 
+The length of time required will depend on the cell line you are using, as they 
+each take a different length of time to divide.
+
+### 2D or 3D
+
+This will depend on how we want to measure nucleus size and shape (as we'll look 
+at in the next section of this episode). For example, if we want to accurately 
+characterise the volume and 3D shape of the nuclei then we will need to image in 
+3D. If we're instead happy to estimate from 2D measures like nucleus diameter, 
+then 2D is sufficient. This again highlights the need for a specific research 
+question! In a real life experiment, we should probably define more clearly 
+exactly what aspects of nucleus size and shape we are interested in, as this 
+will help to inform the need for 2D or 3D.
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Define what you need to measure
+
+Once we have a good idea of what we need to observe, we need to clearly define 
+what measurements are required. 
+
+- What do you need to quantify from your images to answer your question? E.g. 
+cell length, nucleus volume, the number of cells with a specific fluorescent 
+label...
+
+- How will you quantify them? E.g. which image processing software? What kind of 
+analysis methods?
+
+- How many objects/events do you need to quantify?
+
+As the second point indicates, it's always worthwhile to consider how you will 
+quantify your images before you collect them. This will make the image analysis 
+steps much faster and ensure that you are collecting images optimised for the 
+given method. We'll look at image processing methods more in later episodes.
+
+For the last point, it's important to consider any statistical tests you will 
+use to answer your research question, and the sample size required to give them 
+sufficient statistical 'power'. Statistical power is the likelihood of a 
+statistical test detecting an effect when one is actually present. For example, 
+consider testing if the number of cells is significantly different when grown 
+with and without our chemical of interest. If there really is a difference, a 
```

Comment:
```suggestion
consider testing if there is a difference of X% in the number of cells  when grown 
with and without our chemical of interest. If there really is a difference, a 
```
Subtle change - any level of difference can be significant with the right sample size. What we want is a relevant effect size or difference in the number of cells and make sure that we are powered to detect that level of effect size.